packages/components/src/Workflow/_docs/Workflow.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react"
 import { Meta, StoryFn, StoryObj } from "@storybook/react"
 import { Button } from "@kaizen/button"
-import CloseIcon from "@kaizen/component-library/icons/close.icon.svg"
-import VisibleIcon from "@kaizen/component-library/icons/visible.icon.svg"
+import { CloseIcon } from "~icons/CloseIcon"
+import { VisibleIcon } from "~icons/VisibleIcon"
```

Comment:
(OOS) At some point it would be nice if it was like this:
```suggestion
import { CloseIcon, VisibleIcon } from "~icons"
```

Reply:
Looking forward to your PR 😛